src/eterna/ui/ConfirmTargetDialog.ts

```diff
@@ -0,0 +1,45 @@
+import Fonts from 'eterna/util/Fonts';
+import {AlphaTask, HLayoutContainer, VLayoutContainer} from 'flashbang';
+import GameButton from './GameButton';
+import WindowDialog from './WindowDialog';
+
+export default class ConfirmTargetDialog extends WindowDialog<'cancel' | 'reset' | 'submit'> {
+    constructor() {
+        super({title: 'Are you sure?', modal: true});
+    }
+
+    protected added() {
+        super.added();
+
+        const content = new VLayoutContainer(20);
+        this._window.content.addChild(content);
+
+        const PROMPT = 'This design has a custom target structure that will be uploaded along with your design, '
```

Comment:
This is sort of "wall of text" feeling. What do you think of splitting it up a bit?

<img width="441" alt="Screen Shot 2023-04-13 at 1 26 18 PM" src="https://user-images.githubusercontent.com/5233226/231875200-9f6fe68d-277b-42ea-a222-e01c96b48fe4.png">

Reply:
That seems reasonable